dist/textbox/ds6/textbox.css

```diff
@@ -64,6 +64,7 @@ input.textbox__control--floated,
 textarea.textbox__control--floated {
   border: none;
   border-bottom: 1px solid #767676;
+  height: 30px;
   padding: 0;
 }
 input.textbox__control--floated::-webkit-input-placeholder,
```

Comment:
Shouldn't this be `textbox__control--underline` now?

Reply:
Updated.